README.md

```diff
@@ -9,3 +9,36 @@ Slackbot for calculating Harvest balance.
 * Supports only Finnish working regulation and public holidays.
 
 Slackbot is deployed in Google Cloud, partial terraform configurations can be found in infra-folder.
+
+# Installation & running
+
+Get Harvest access token from your harvest account page under Developers.
+
+Get GOOGLE_APPLICATION_CREDENTIALS from here: https://console.cloud.google.com/apis/credentials/serviceaccountkey
+
+* From the Service account list, select New service account.
+* In the Service account name field, enter a name.
+* From the Role list, select Project > Owner.
```

Comment:
Since this is something common for all GCloud projects, can we just add a link here to instructions how to start developing with GCloud e.g. https://cloud.google.com/docs/authentication/getting-started